content/tutorials/wallets.md

```diff
@@ -1,52 +1,66 @@
+We will explain briefly what Metamask is, and how to setup a Metamask account.
 
 **What is MetaMask?**
-MetaMask is a browser extension that allows web applications to interact with the Ethereum blockchain. In our current setup, browsers like Chrome/Firefox display information by fetching it from a database. Our current web browsers (web 2.0 as it’s called) are not build to interface with distributed databases. This is why MetaMask is needed, as it allows modern web browsers to interface with the Ethereum blockchain.
+MetaMask is a browser extension that allows web applications to interact with the Ethereum blockchain. In our current setup, browsers like Chrome/Firefox display information by fetching it from a server. Our current web browsers (web 2.0 as it’s called) are not build to interface with distributed systems. This is why MetaMask is needed, as it allows modern web browsers to interact with the Ethereum blockchain.
+
 
 **Why is Metamask required?**
 The unique part about MetaMask is that it serves a dual purpose as an ERC-20 wallet and a Web 3.0 browser. For users, it works as an Ethereum wallet, allowing them to store and send any ERC-20 tokens. For developers, it allows you to design and run Ethereum DApps right in your browser without running a full Ethereum node. MetaMask talks to the Ethereum blockchain for you.
 
 **How to setup Metamask?**
-Here are some setup Instructions for Metamask:
+Here are some setup Instructions for Metamask
 
 1. Go to the Chrome Web Store for extensions and search/install metamask. ![metamask-chrome-store](images/metamask-chrome-extension.png)
-2. It will get added as a browser extension on the top right portion. Go ahead and accept the terms and conditions. Create a username and password as well. ![tandc-metamask](images/metamask-create-username-password.png)
+2. It will get added as a browser extension on the top right portion. Read through and accept the terms and conditions. Create a username and password in the next step. ![tandc-metamask](images/metamask-create-username-password.png)
 3. MetaMask will generate a secret backup phrase for you. Write it down, store it in a safe place, and click next. ![backup](images/metamask-secret-passcode.png)
 4. Confirm your secret backup phrase and finish your MetaMask wallet set up! ![setup-final-metamask](images/metamask_view-account-options.png)
-5. Voila! Your account is now created! You can now buy and sell tokens. You can copy your account address to clipboard from the options. This address is where you will be receiving all your ERC-20 tokens.
+5. Voila! Your account is now created! You can now store tokens in your wallet. You can copy your account address to clipboard from the options. This address is where you will be receiving all your ERC-20 tokens.  
 
-You can find more resources for setting up [here](https://drive.google.com/open?id=1N68MsRlbxJ4h6otCaTIwN4y904unh1QO).
+You can find more resources for setting up [here]().
```

Comment:
As a general rule: never use the word `here` as a link title, always use something descriptive like `...for setting up in this [installation video]()`.

second thing: it's linking to nothing. You might want to take a look at embedding videos from various providers: https://docs.oceanprotocol.com/test/#videos

Reply:
I was going to add the link there later, once I created a youtube playlist as Sarah was uploading it on YouTube. But I've linked the videos in the tutorial steps instead.